docs/tutorials/ui-customization.md

```diff
@@ -96,6 +96,7 @@ The following buttons can be added to the overflow menu:
 * remote: adds a button that opens a Remote Playback dialog. The button is visible only if the
   browser supports Remote Playback API.
 * Statistics: adds a button that displays statistics of the video.
+* displayChapters: tick marks between chapters and labels that appear on hover.
```

Comment:
This configuration does not have to exist, in the same way as we do with thumbnails. Please, remove this config.